elliottlib/cli/attach_bugs_cli.py

```diff
@@ -84,7 +84,7 @@ def attach_bugs_cli(runtime: Runtime, advisory, default_advisory_type, bug_ids,
         advisory = find_default_advisory(runtime, default_advisory_type)
 
     bug_ids = cli_opts.id_convert_str(bug_ids)
-    bugs = bug_tracker.get_bugs(bug_ids, verbose=runtime.debug)
+    bugs = bug_tracker.get_bugs(bug_ids)
 
     # Check if target release and OCP version match
     target_release = Bug.get_target_release(bugs)
```

Comment:
BugzillaBug need to have target_release, maybe `self.target_release = self.bug.target_release` in init like jiraBug?

Reply:
I added in https://github.com/openshift/elliott/pull/305/files#diff-7863de3d0ccba5169f0ec377e6f39f5dcdd73322fc35d9a8a5b9fbe4ca1cc28dR86